app/src/components/v-checkbox.vue

```diff
@@ -143,7 +143,7 @@ body {
 			width: 100%;
 			background-color: transparent;
 			border: none;
-			border-bottom: 2px solid var(--border-normal);
+			border-bottom: 2px solid var(--theme--form--field--input--border-color);
```

Comment:
Couldn't we use the var here as well?
```suggestion
			border-bottom: var(--theme--border-width) solid var(--theme--form--field--input--border-color);
```